### ISSUE TITLE
Warn when CLI and Server are out of sync

### ISSUE THREAD

**Issue by timsuchanek** [prisma/prisma#3991]
When `prisma deploy` is executed and the server and cli version have a different minor version (a different x in 1.x), the CLI should print a warning that this may result in unwanted behavior.

**Comment by divyenduz** [comment]
Warning with yellow text can be emitted as 

```
Warning: your Prisma server and Prisma CLI versions are out of sync, they should be on the same minor version. 

Prisma CLI version: 1.x
Prisma server version: 1.y
```

Here, x, y would be filled dynamically.

**Comment by pantharshit00** [comment]
I also think it can be handy to prisma the `npm install` command for the CLI to sync it.

Also, this implies we should remove the version upgrade check.

**Comment by mavilein** [comment]
In my opinion it should not be possible to disable this warning.

**Comment by divyenduz** [comment]
This RFC https://github.com/prisma/rfcs/pull/9 has the detailed discussion around this feature.

**Comment by divyenduz** [comment]
This is available in the latest stable `1.31.1`. Please install this via `npm install -g prisma`.

I am closing this issue, please open a new issue again if necessary.
